lib/timeline/timeline.dart

```diff
@@ -260,21 +260,21 @@ class TimelineFrameUI extends CoreElement {
     bool isSlow = false;
 
     final CoreElement dartBar = div(c: 'perf-bar left');
```

Comment:
add TODO to add a helper so we aren't calling this same logic twice. 
Helper should draw a bar given a duration and a style (assuming we want slightly different styles in the future for gpu vs cpu, etc).

Reply:
done.